content/guides/models/track/log/distributed-training.md

```diff
@@ -45,22 +51,24 @@ if __name__ == "__main__":
         train(args)
 ```
 
-Explore the W&B App UI to view an [example dashboard](https://wandb.ai/ayush-thakur/DDP/runs/1s56u3hc/system) of metrics tracked from a single process. The dashboard displays system metrics such as temperature and utilization, that were tracked for both GPUs.
+Explore this example dashboard in the W&B App UI to [view metrics tracked from a single process](https://wandb.ai/ayush-thakur/DDP/runs/1s56u3hc/system). The dashboard displays system metrics for both GPUs, such as temperature and utilization.
```

Comment:
```suggestion
Explore an [example dashboard showing metrics tracked from a single process](https://wandb.ai/ayush-thakur/DDP/runs/1s56u3hc/system).

The dashboard displays system metrics for both GPUs, such as temperature and utilization.
```
Splitting this up helps with the "However" sentence below and with the interspersed images.